session/create_producer_test.go

```diff
@@ -47,8 +47,9 @@ func TestProducer_RequestSessionCreate(t *testing.T) {
 	sender := &fakeSender{}
 	sid, config, err := RequestSessionCreate(sender, 123, []byte{}, ConsumerInfo{})
```

Comment:
I guess `sid` was for `sessionID`, but now it looks strange: `sid.ID`.

Reply:
And looks like it's not a `config` anymore?

Reply:
changed both